### ISSUE TITLE
No data in sFlow graph

### ISSUE THREAD

**Issue by jensmolsson** [ntop/ntopng#1977]
After update to version shown below sFlow graph unit stopped to work - Just showing "no data avaible".

[root@noptr ~]# ntopng --version
v.3.7.180903    [Enterprise/Professional build]

URL:
/flowdevices_stats.lua?ip=[IP]&page=historical&sflow_filter=All

**Comment by simonemainardi** [comment]
Ok, we've looked into this issue.

There were several permission errors on the system. Very likely you moved the ntopng data directory to /etc/ntopng/data without changing the permissions as highlighted in https://www.ntop.org/support/faq/migrate-the-data-directory-in-ntopng/

So now we' ve moved the data dir to the recommended `/var/lib/ntopng` and we've proposed a fix for the missing charts. A new build is in progress. Hold on an hour and upgrade ntopng to the latest 3.7 dev.
